docs/admin/cluster-large.md

```diff
@@ -40,35 +40,63 @@ To avoid running into cloud provider quota issues, when creating a cluster with
     * Target pools
 * Gating the setup script so that it brings up new node VMs in smaller batches with waits in between, because some cloud providers rate limit the creation of VMs.
 
-### Etcd storage
-
-To improve performance of large clusters, we store events in a separate dedicated etcd instance.
-
-When creating a cluster, existing salt scripts:
-
-* start and configure additional etcd instance
-* configure api-server to use it for storing events
-
+### Etcd storage
+
+To improve performance of large clusters, we store events in a separate dedicated etcd instance.
+
+When creating a cluster, existing salt scripts:
+
+* start and configure additional etcd instance
+* configure api-server to use it for storing events
+
+### Size of master and master components
+
+On GCE/GKE and AWS, `kube-up` automatically configures the proper VM size for your master depending on the number of nodes
+in your cluster. On other providers, you will need to configure it manually. For reference, the sizes we use on GCE are
+* 1-5 nodes: n1-standard-1
+* 6-10 nodes: n1-standard-2
+* 11-100 nodes: n1-standard-4
+* 101-250 nodes: n1-standard-8
+* 251-500 nodes: n1-standard-16
+* more than 500 nodes: n1-standard-32
+
+And the sizes we use on AWS are
+* 1-149 nodes: m3.medium
+* more than 150 nodes: m3.large
+
+Note that these master node sizes are currently only set at cluster startup time, and are not adjusted if you later scale your cluster up or down.
+
+We have measured the following 99th percentile resource consumption for the various master components on GCE at
+(100 nodes/500 nodes/1000 nodes) cluster size:
+* main etcd instance: CPU (0.135/0.241/0.312) RAM MB (242/706/1792)
+* events etcd instance: CPU (0.082/0.099/0.095) RAM MB (237/746/1846)
+* API server: CPU (1.233/3.0/5.9) RAM MB (895/2439/4285)
+* controller manager: CPU (0.345/0.633/3.57) RAM MB (320/1624/4507)
+* scheduler: CPU (0.232/1.576/1.758) RAM MB (48/167/5541)

```

Comment:
I'm wondering about giving those numbers.
From these number you may think that having 16-core master in 1000-node cluster is definitely enough. However, with that size of master it happens that we are cpu-bounded, which in turns causes violating our SLOs.
So maybe not showing them is a better idea for now?